### DISCUSSION TITLE
How can I find the course to learn about the PK-sim software

### DISCUSSION THREAD

**Discussion by Zhaowenyu-gaga** [Open-Systems-Pharmacology/Forum#848] [Questions & Problems] [answered]
I am a novice for the knowledge about the PBPK model and found this software in this platform.

And my project is about an organic compound's metabolism in human body. We just want to observe the change of this compound in different organ under various time level. Could I use this software to get that? And how can I learn this software? I have downloaded this software but I did not find how to add my parameters

Thank you very much for your help. I am in such perplexed.

**Comment by StephanSchaller** [comment, marked as answer]
Please refer to the tutorials e.g. here:

http://www.open-systems-pharmacology.org/#tutorials

and some videos here:

https://www.youtube.com/channel/UCrjhCoM0g5C5wtQQ4JQYpJg/

**Comment by Zhaowenyu-gaga** [comment]
Thank you very much for your answer, it really helped me.

By the way, could you please guide me whether this software can simulate the quantity of one chemical compound's residue in certain organs for animals after a few days' exposure, such as in the chicken (I did not find "chicken" option in the "individuals" part).

Thank you for your help in advance.

**Comment by StephanSchaller** [comment]
There are a number of different animals you can select from although chicken is currently not one of them.

A Chicken PBPK model is currently in development and will be released likely in apring/early summer 2021.

**Comment by Yuri05** [comment]
>A Chicken PBPK model is currently in development and will be released likely in apring/early summer 2021.

cool :)
<img width="150" src="https://user-images.githubusercontent.com/25061876/98053108-22749680-1e38-11eb-8989-7dc546d1080b.png">

**Reply by abbakardvm** [reply]
Hello Sir, 
I write with respect to your answer about PBPK model for chickens whether its will be available this year 2021 or the project will get in to completion only in 2022. The reason why I inquired about it is that I'm doing my research in laying hens and wanted to use PK-sim to build a model for estimating drug withdrawal time in eggs after exposure. Therefore I need your assistant to guide me through. 
Thank you.

**Reply by abbakardvm** [reply]
> > A Chicken PBPK model is currently in development and will be released likely in apring/early summer 2021.
> 
> cool :) <img alt="" width="150" src="https://user-images.githubusercontent.com/25061876/98053108-22749680-1e38-11eb-8989-7dc546d1080b.png">

Hello Sir,
I write with respect to your answer about PBPK model for chickens whether its will be available this year 2021 or the project will get in to completion only in 2022. The reason why I inquired about it is that I'm doing my research in laying hens and wanted to use PK-sim to build a model for estimating drug withdrawal time in eggs after exposure. Therefore I need your assistant to guide me through.
Thank you.

**Reply by StephanSchaller** [reply]
Hi @abbakardvm , yes, sorry to say, that the release of the chicken model will be delayed and available only in (early) 2022.

**Reply by abbakardvm** [reply]
@StephanSchaller  Sir, 
Thanks for your prompt reply. Moreover, I would like to ask if your organization would allow offering internship training (not necessarily mean paid one, rather take the responsibility of myself) for a Ph.D. candidate so that one can learn and master the skills which would be used for analyzing his research data.  I would be grateful if you would grant me such an opportunity. 
Thank you.

**Reply by abbakardvm** [reply]
Hello Stephan,

I hope my text get you well. Please I would like to contact you again and to enquire about the **chicken PBPK** model. Please should we expect it release this year or still it may be delayed for another time? 
Kind regards

**Reply by StephanSchaller** [reply]
@abbakardvm, we are currently finalizing the publication for submission, so I am confident in will be released before end Q2 2022.

**Reply by abbakardvm** [reply]
@StephanSchaller , thank you for the good news. I'm happy that you succeeded in attaining this feat.
